src/test/java/org/mule/service/scheduler/internal/DefaultSchedulerQuartzTestCase.java

```diff
@@ -184,7 +186,10 @@ public void cronExceeds() throws SchedulerException {
     verify(sharedQuartzScheduler).scheduleJob(any(JobDetail.class), argThat(new CronTriggerMatcher(everySecond)));
 
     new PollingProber().check(new JUnitLambdaProbe(() -> {
+      assertThat(startTimes.size(), is(3));
+      assertThat(endTimes.size(), is(3));
       assertThat((double) NANOSECONDS.toMillis(startTimes.get(2) - endTimes.get(1)), closeTo(0, DELTA_MILLIS));
+      assertThat((double) NANOSECONDS.toMillis(startTimes.get(1) - endTimes.get(0)), closeTo(0, DELTA_MILLIS));
```

Comment:
Monitor this in case it happens what was said regarding the lack of stability in the first comparison,